sdk/monitor/monitor-query/tsconfig.json

```diff
@@ -0,0 +1,11 @@
+{
+  "extends": "../../../tsconfig.package",
+  "compilerOptions": {
+    "outDir": "./dist-esm",
+    "declarationDir": "./types",
+    "paths": {
+      "@azure/template": ["./src/index"]
```

Comment:
👀

Reply:
Also fixed (and became more obvious once I got my samples into samples-dev)